core/src/main/scala/io/finch/EndpointStreamModule.scala

```diff
@@ -0,0 +1,26 @@
+package io.finch
+
+import cats.effect.Effect
+import com.twitter.io.Buf
+import io.finch.streaming.{DecodeStream, StreamFromReader}
+
+trait EndpointStreamModule[S[_[_], _], F[_]] extends EndpointModule[F] {
```

Comment:
I'm starting to think this could be an overkill at this point. I'd imagine the following still works (please, correct me if I'm wrong):

```
import io.finch.catsEffect._
import io.finch.fs2._

val e = streamBinaryBody[fs2.Stream]
```

I know I suggested that `StreamModule` idea in the first place. After Thinking about this more, the types seem less scary now, even if we use just a bare bones `EndpointModule`. What do you think?

Reply:
Removed